### PR TITLE
Add incomplete penetrance indication to the report

### PR DESCRIPTION
**Pull request by bartcharbon**

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -77,5 +78,6 @@
   "genomeBrowserUnavailable": "Genome browser unavailable for selected record.",
   "treeViewer": "Decision tree",
   "missingDecisionTree": "Decision tree unavailable",
-  "treeInfo": "Click and drag in the tree to navigate, scroll to zoom."
+  "treeInfo": "Click and drag in the tree to navigate, scroll to zoom.",
+  "incompletePenetrance": "Gene known to show incomplete penetrance."
```
Comment (dennishendriksen) [inline src/locales/en.json:82]: known to show --> associated with

**Review by dennishendriksen — CHANGES_REQUESTED** [review]
Values in the literature column are rendered incorrectly:
template_issue
This is not the case on the master branch.

**Comment by dennishendriksen** [comment]
how about updating apiDataMock.ts so that a gene with incompletePenetrance is included?

File: src/components/RecordInfoDetailsItem.vue [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+      <Gene v-if="value.length > 0" :value="value" />
+    </span>
     <RecordInfoDetailsItemValue v-if="value !== null" :metadataId="metadata.id" :value="value" :details="details" />
   </span>
   <span v-else>
     <span v-if="metadata.id === 'PUBMED'">
       <PubMedAnchor v-if="value.length > 0" :pubMedIds="value" />
     </span>
+    <span v-if="metadata.id === 'GeneObject'">
+      test2<Gene v-if="value.length > 0" :value="value" />
```
[the remark this answers is not in the document]
Comment (bartcharbon) [reply src/components/RecordInfoDetailsItem.vue]: oops, that was, well... a test...

File: package-lock.json [hunk 3]
```diff
@@ -105,6 +105,10 @@
       },
       "engines": {
         "node": ">=6.9.0"
+      },
```
[the remark this answers is not in the document]
Comment (bartcharbon) [reply package-lock.json]: not sure, reverted it and stuff still looks good, zo not necessary I guess

**Pull request merged**